composer.json

```diff
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=7.0.0",
         "fideloper/proxy": "~3.3",
+        "guzzlehttp/guzzle": "^6.3",
```

Comment:
What is Guzzle being used for?

Reply:
I believe it's for interacting with the SMTP server

Reply:
I think it's added automatically via the php artisan make:mailable something